src/sage/combinat/sloane_functions.py

```diff
@@ -859,7 +859,7 @@ def _eval(self, n):
             sage: [sloane.A003418._eval(n) for n in range(1,11)]
             [1, 2, 6, 12, 60, 60, 420, 840, 2520, 2520]
         """
-        return arith.lcm([i for i in range(1, n+1)])
+        return arith.lcm(list(range(1, n + 1)))
```

Comment:
Shouldn't we rather fix the documentation of `sage.arith.functions.lcm` to clarify that the argument can be any iterable, not just a list or tuple?